addons/website/static/src/js/website.js

```diff
@@ -365,6 +365,9 @@ dom_ready.then(function () {
         init_kanban(this);
     });
 
+    $('.js_submit_loading').on('submit',function (){
+        $(this).find('#btn-loading').button('loading');
```

Comment:
Don't put an id on each button : you can find it with `$(this).find("button[type='submit']")`.
Then extract the actual text of the button, and update its content with something like `.html($('<span><i class="fa fa-spinner fa-spin"/> '+text+'</span>'));` Also, disabled the button in this function.
The form should only have the new class on them.